src/property.html

```diff
@@ -0,0 +1,1066 @@
+                  <p
+                    class="text-lg sm:text-xl font-medium text-color-addition-info"
+                  >
+                    <span class="text-color">01</span> of 10
+                  </p>
+                </div>
+                <div class="flex gap-2 justify-between items-center">
+                  <img
+                    src="/public/assets/propertypage/left.svg"
+                    alt="left-button"
+                    class="w-11 h-11 border-button p-3.5 rounded-full"
+                  />
+                  <p
+                    class="text-sm sm:text-xl font-medium text-color-addition-info xl:hidden"
+                  >
+                    <span class="text-color">01</span> of 60
+                  </p>
+                  <img
+                    src="/public/assets/propertypage/right.svg"
+                    alt="right-button"
+                    class="w-11 h-11 border-button p-3.5 bg-for-logo-section rounded-full"
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+        <!-- section personal info details -->
+        <section class="px-3 sm:px-8 xl:px-10 2xl:px-20">
+          <img
+            src="/public/assets/propertypage/pricing-logo-star.png"
+            alt="logo"
+          />
+          <div class="mt-1 ml-2 flex flex-col gap-10 lg:gap-12 2xl:gap-20">
+            <!-- heading -->
+            <div class="grid lg:grid-cols-6 gap-2">
+              <p
+                class="font-semibold lg:col-span-6 text-2xl xl:text-4xl text-color"
+              >
+                Let's Make it Happen
+              </p>
+              <p
+                class="font-medium lg:col-span-4 text-sm sm:text-base 2xl:text-lg text-color-addition-info"
+              >
+                Ready to take the first step toward your dream property? Fill
+                out the form below, and our real estate wizards will work their
+                magic to find your perfect match. Don't wait; let's embark on
+                this exciting journey together.
+              </p>
+            </div>
+            <!-- full form and info -->
+            <div
+              class="rounded-xl border-button flex flex-col gap-8 2xl:gap-12 p-5 lg:p-12 2xl:p-24"
+            >
+              <!-- for form page -->
+              <div
+                class="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4 gap-5 lg:gap-8 2xl:gap-12"
+              >
+                <!-- first -->
+                <div
+                  class="flex flex-col col-span-2 sm:col-span-1 gap-2.5 lg:gap-3.5 2xl:gap-4"
+                >
+                  <p class="font-semibold text-base 2xl:text-xl text-color">
+                    First Name
+                  </p>
+                  <p
+                    class="rounded-lg font-medium text-sm 2xl:text-lg leading-5 border-button px-5 py-4 2xl:px-5 2xl:py-6 bg-for-logo-section text-color-addition-info"
```

Comment:
will not mention it again, need to change it everywhere

Reply:
but why its a cool name i also give it to every para in every file

Reply:
brother, you are doing shit, and even arguing. If I am saying a thing should not like that, that must not be. Please change

Reply:
ok

Reply:
really vs code searcher is really a good thing just one click and change every thing

Reply:
@hassnainzia121 is it fixed?

Reply:
yes

Reply:
remove all vanilla custom css and write it in tailwindcss and config file